skaffold.yaml

```diff
@@ -0,0 +1,115 @@
+---
+apiVersion: skaffold/v4beta7
+kind: Config
+metadata:
+  name: gcmx
```

Comment:
Can we pick a more descriptive name?

Reply:
I'll defer this to your judgement, as I have no strong feelings about it. The overall naming approach I'm using is:
- User audiences: 'gardener-custom-metrics'
- Developer audiences: 'GCMx' and 'gcmx'

However, it is important to view this name in the light that 'GCMx' will be the official abbreviation for gardener-custom-metrics. The exposure of the abbreviations GCMx and BIPA should be on par with e.g. the abbreviation 'HVPA'.

Reply:
Renamed to 'gardener-custom-metrics'